docs/mnist_example/requirements.txt

```diff
@@ -1,2 +1,3 @@
-torch~=1.13.0
-torchvision~=1.14.0
+torch~=1.13.1
+torchvision~=0.14.1
+pillow~=9.3.0
```

Comment:
The list is small now, but to keep things easy in the future, can the list be put in alphabetical order?

Also add:
```
numpy~=1.22.1
```

Reply:
Done